python/grass/jupyter/interactivemap.py

```diff
@@ -346,6 +347,110 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def setup_drawing_interface(self):
+        """
+        Allow Users to draw Geometry and Save them.
+        """
+        import ipywidgets as widgets
+        from IPython.display import display
+        import tempfile
+
+        draw_control = self._ipyleaflet.DrawControl()
```

Comment:
I mentioned this before, but here it is again from ipyleaflet documentation:
_The DrawControl is deprecated and will be removed in a future release. Please use GeomanDrawControl instead._

Reply:
Hi Anna,

Sorry, it slipped out of my mind. I have made the changes and cannot hide the controls. Could you please help me with this?